### ISSUE TITLE
Screen reader announces every character twice when user types in ‘Add new feedback’ edit field in Windows.

### ISSUE THREAD

**Issue by msftedad** [microsoft/vsts-extension-retrospectives#1261]
#accessibility #A11y

**Describe the bug**

Screen reader announces every character twice when user types in ‘Add new feedback’ edit field in Windows. Screen reader users experience confusion and increased cognitive load as each character they type is announced twice. 

**Are you using the Retrospectives Tool on-prem?**
NO

**To Reproduce**

1. Open the URL: [Retrospectives - Boards](https://dev.azure.com/eDADPlayground/SCRUMProject/_apps/hub/ms-devlabs.team-retrospectives.home#teamId=1994b7b8-6d1b-4c29-9d29-46c65d6657cd&boardId=ef28c6ea-2637-490e-86fa-a7b24f59a7ec) in latest edge browser. 
2. Retrospectives’ page will appear. 
3. Press tab key to reach to the ‘Collect’ tab and activate it using enter key. 
4. Press tab key to ‘Add new feedback’ control and activate it using enter key.  
5. Feedback card with ‘edit field’ will appear. Press tab key to reach to it and enter some text.  
6. Verify whether screen reader announces every character twice when user types in ‘Add new feedback’ edit field or not.

**Expected behavior**
Screen reader shouldn’t announce every character user types twice in ‘Add new feedback’ edit field in Windows.  When a character is entered in Edit field, it should announce only once.

**Screenshots**

https://github.com/user-attachments/assets/4e02e2e3-ee81-4029-b41f-142ac264b6d8

**What screen type are you seeing this on ?**
Desktop Windows 11

**Additional context**
